.github/workflows/javascript-lint.yml

```diff
@@ -4,12 +4,18 @@ on:
   push:
     paths:
       - '**/*.js'
+      - '**/*.cjs'
+      - '**/*.mjs'
       - '**/*.json'
+      - 'eslint/'
```

Comment:
Feels redundant? Only .json and .js files in there?

Reply:
Thanks, good point!